client/my-sites/pages/page.jsx

```diff
@@ -324,9 +322,9 @@ const Page = React.createClass( {
 	},
 
 	popoverMoreInfo: function() {
```

Comment:
@jblz or anyone having worked with you: would you know the exact steps / site state needed so that I can see this piece of UI rendered on my Calypso instance? This is so I could determine whether the move to `createFragment` would also help here.